tests/cache/tests.py

```diff
@@ -427,6 +431,26 @@ def test_expiration(self):
         self.assertEqual(cache.get("expire2"), "newvalue")
         self.assertFalse(cache.has_key("expire3"))
 
+    def test_touch(self):
+        # prolong cache (explicit timeout)
+        cache.set('expire1', 'very quickly', timeout=1)
+        cache.touch('expire1', timeout=4)
+        time.sleep(2)
+        self.assertTrue(cache.has_key('expire1'))
+        time.sleep(3)
+        self.assertFalse(cache.has_key('expire1'))
+
+        # prolong cache (implicit timeout)
+        cache.set('expire1', 'very quickly', timeout=1)
+        cache.touch('expire1')
+        time.sleep(2)
+        self.assertTrue(cache.has_key('expire1'))
+
+        # return value
+        cache.set('something', 'else')
+        self.assertTrue(cache.touch('something'))
```

Comment:
Use `self.assertIs(cache.touch('something'), True)` since `assertTrue()` passes if `bool(value) is True`.